### PR TITLE
fix some doc styles for files in doc/

### PR DESCRIPTION
**Pull request by dabue**
fix some doc styles by markdownlint

related issue: #1273

### CODE REVIEW

File: doc/admin-api.md [hunk 1]
```diff
@@ -17,8 +17,8 @@
 #
 -->
 
+# Table of contents
```
Comment (juzhiyuan) [inline doc/admin-api.md:20]: Please use Contents instead of contents

File: doc/router-radixtree.md [hunk 2]
```diff
@@ -32,15 +32,15 @@ Let's take a look at a few examples and have an intuitive understanding.
 
 #### 1. Full match
 
-```
+```uri
```
Comment (juzhiyuan) [inline doc/router-radixtree.md:35]: URI is not a valid code syntax decorator as I know...

File: doc/stream-proxy-cn.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ APISIX 可以对 TCP/UDP 协议进行代理并实现动态负载均衡。 在 ng
 
 在 `conf/config.yaml` 配置文件设置 `stream_proxy` 选项， 指定一组需要进行动态代理的IP地址。默认情况不开启stream代理。
 
-```
+```config
```
Comment (juzhiyuan) [inline doc/stream-proxy-cn.md:33]: Please check this decorator, too.

File: doc/stream-proxy.md [hunk 4]
```diff
@@ -30,7 +30,7 @@ APISIX can dynamic load balancing TCP/UDP proxy. In Nginx world, we call TCP/UDP
 Setting the `stream_proxy` option in `conf/config.yaml`, specify a list of addresses that require dynamic proxy.
 By default, no any stream proxy is enabled.
 
-```
+```config
```
Comment (juzhiyuan) [inline doc/stream-proxy.md:33]: ,

**Comment by dabue** [comment]
thx， I' ll check it.



88786912@qq.com
 
From: 琚致远
Date: 2020-04-19 19:45
To: apache/incubator-apisix
CC: dabue; Author
Subject: Re: [apache/incubator-apisix] fix some doc styles for files in doc/ (#1475)
@juzhiyuan requested changes on this pull request.


In doc/admin-api.md:
> @@ -17,8 +17,8 @@
 #
 -->
 
+# Table of contents

Please use Contents instead of contents


In doc/router-radixtree.md:
> @@ -32,15 +32,15 @@ Let's take a look at a few examples and have an intuitive understanding.
 
 #### 1. Full match
 
-```
+```uri

URI is not a valid code syntax decorator as I know...


In doc/stream-proxy-cn.md:
> @@ -30,7 +30,7 @@ APISIX 可以对 TCP/UDP 协议进行代理并实现动态负载均衡。 在 ng
 
 在 `conf/config.yaml` 配置文件设置 `stream_proxy` 选项， 指定一组需要进行动态代理的IP地址。默认情况不开启stream代理。
 
-```
+```config

Please check this decorator, too.


In doc/stream-proxy.md:
> @@ -30,7 +30,7 @@ APISIX can dynamic load balancing TCP/UDP proxy. In Nginx world, we call TCP/UDP
 Setting the `stream_proxy` option in `conf/config.yaml`, specify a list of addresses that require dynamic proxy.
 By default, no any stream proxy is enabled.
 
-```
+```config

,

**Comment by dabue** [comment]
I have updated it.

File: doc/plugins/limit-count-cn.md [hunk 5]
```diff
@@ -123,7 +123,7 @@ Server: APISIX web server
 
 当你第三次访问的时候，就会收到包含 503 返回码的响应头：
 
-```console
```
Comment (juzhiyuan) [inline doc/plugins/limit-count-cn.md]: `console` is a valid decorator actually 😂

**Comment by membphis** [comment]
many thx, merged

**Pull request merged**